src/ResourceManager/LogicApp/Commands.LogicApp/Cmdlets/IntegrationAccount/GetAzureIntegrationAccountReceivedIcnCommand.cs

```diff
@@ -65,11 +75,17 @@ public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
 
+            if (string.IsNullOrEmpty(AgreementType))
+            {
+                this.WriteWarning("By default, you are using the X12 agreement. Please provide a value for AgreementType if you would like to specify the agreement type. Possible values are X12 and Edifact.");
```

Comment:
I am wondering if resource file can be used in cmdlet projects instead of copying the same string around.